### PR TITLE
[2221] add logging to logit

### PR DESCRIPTION
**Pull request by misaka**
### Context

Applications Insights is insufficient.

### Changes proposed in this pull request

Send our logs to logit.io

### Guidance to review

✂️ 🌲 🌲 🌲 ✂️

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -244,6 +256,14 @@
                 "name": "WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION",
                 "value": "0"
               }
```
Comment (rizzkhan1) [inline azure/template.json]: missing a comma here
```suggestion
              },
```

Comment (misaka) [reply azure/template.json]: :doh:

File: azure/template.json [hunk 2]
```diff
@@ -244,6 +256,14 @@
...
+              {
+                "name": "SETTINGS__LOGSTASH__HOST",
+                "value": "[parameters('logstashHost')]"
```
Comment (rizzkhan1) [inline azure/template.json]:
```suggestion
                "secureValue": "[parameters('logstashHost')]"
```

Comment (misaka) [reply azure/template.json]: Is this needed for app servers too? For some reason I thought it was only container instances.

File: azure/template.json [hunk 3]
```diff
@@ -118,7 +118,19 @@
...
+      "metadata": {
+        "description": "The host name of the logstash server."
+      }
+    },
+    "logstashPort": {
+      "type": "string",
+      "metadata": {
+        "description": "The port number for the logstash server connection."
+      }
+    },
```
Comment (misaka) [inline azure/template.json]: @rizzkhan1 Is the trailing comma here a problem?

Comment (rizzkhan1) [reply azure/template.json]: Yes, You don't need a comma in line 133

Comment (misaka) [reply azure/template.json]: Fixed.

**Pull request merged**